src/Features/LanguageServer/Protocol/Features/Options/WorkspaceConfigurationOptionsStorage.cs

```diff
@@ -36,4 +40,15 @@ public static WorkspaceConfigurationOptions GetWorkspaceConfigurationOptions(thi
 
     public static readonly Option2<bool> EnableOpeningSourceGeneratedFilesInWorkspaceFeatureFlag = new(
         "dotnet_enable_opening_source_generated_files_in_workspace_feature_flag", WorkspaceConfigurationOptions.Default.EnableOpeningSourceGeneratedFiles);
+
+    public static readonly Option2<SourceGeneratorExecutionPreference?> SourceGeneratorExecution = new(
+        "dotnet_source_generator_execution",
+        defaultValue: null,
+        isEditorConfigOption: true,
```

Comment:
How does a user override this when one person on the team has a machine that performs poorly, and others do not?

Reply:
Thats how this option works.  It uses standard roaming storage (stored at `new RoamingProfileStorage("TextEditor.Roslyn.Specific.SourceGeneratorExecution")`) so that a local setting takes precedence over a stored editor config setting.